package.json

```diff
@@ -6,10 +6,12 @@
   "iife": "dist/global-nav.js",
   "scripts": {
     "build": "rm -rf dist && rollup -c",
-    "lint": "yarn sass-lint src/**/*.scss -v && yarn eslint src/js",
+    "serve": "http-server -p 8300",
+    "test": "yarn sass-lint src/**/*.scss -v && yarn eslint src/js",
```

Comment:
As described, it is useful to consider and test linting separately from generic "test"ing. So I would leave the `"lint"` script, include `prettier **/* --check` in the `"lint"` script, and then make the `"test"` one be simply `yarn run lint`.